client/task-list/tasks/payments/RemotePayments/components/BacsPaymentGatewaySetup.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import {
+	Form,
+	H,
+	TextControl,
+	WooRemotePayment,
+} from '@woocommerce/components';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+import { registerPlugin } from '@wordpress/plugins';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+const initialFormValues = {
+	account_name: '',
+	account_number: '',
+	bank_name: '',
+	sort_code: '',
+	iban: '',
+	bic: '',
+};
+
+const BacsPaymentGatewaySetup = () => {
+	const isUpdating = useSelect( ( select ) => {
+		return select( OPTIONS_STORE_NAME ).isOptionsUpdating();
+	} );
+	const { createNotice } = useDispatch( 'core/notices' );
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+
+	const validate = ( values ) => {
+		const errors = {};
+
+		if ( ! values.account_number && ! values.iban ) {
+			errors.account_number = errors.iban = __(
+				'Please enter an account number or IBAN',
+				'woocommerce-admin'
+			);
+		}
+
+		return errors;
+	};
+
+	const updateSettings = async ( values, markConfigured ) => {
+		const update = await updateOptions( {
+			woocommerce_bacs_settings: {
+				enabled: 'yes',
+			},
+			woocommerce_bacs_accounts: [ values ],
+		} );
+
+		if ( update.success ) {
+			markConfigured();
+			createNotice(
+				'success',
+				__(
+					'Direct bank transfer details added successfully',
+					'woocommerce-admin'
+				)
+			);
+		} else {
+			createNotice(
+				'error',
+				__(
+					'There was a problem saving your payment settings',
+					'woocommerce-admin'
+				)
+			);
+		}
+	};
+
+	return (
+		<>
+			<WooRemotePayment id="bacs">
```

Comment:
Are you using `<WooRemotePayment />` instead of `<WooRemotePaymentForm />` just because there is no plugin installation required here? Just a curiosity, since I'd wondered about use-cases for the more general slot.

Reply:
Yup!  Did you have a different idea around how to address those without plugin installers and steppers?  I suppose we could check for the number of steps and render a `Stepper` if >=2 steps are present if we wanted to remove this slot.